### PR TITLE
Make RSpec/ExampleWording handle "it will" future tense

### LINKED ISSUE #1751: RSpec/ExampleWording: autocorrect "it will" (future tense) to the present tense
From https://rspec.rubystyle.guide/#should-in-example-docstrings

> The descriptions represent actual functionality, not what might be happening. Use the third person in the present tense.

However, I sometimes see examples written in the future tense:

```
it 'will return nil'

it 'will not return nil'
```

Just like misusing "should", this could emit an offense or be autocorrected to the present tense:

```
it 'returns nil'

it 'does not return nil'
```

### PR DESCRIPTION
**Pull request by jdufresne**
Fixes #1751

**Replace this text with a summary of the changes in your PR. The more detailed you are, the better.**

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [x] Added tests.
- [x] Updated documentation.
- [x] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

If you have modified an existing cop's configuration options:

- [x] Set `VersionChanged: "<<next>>"` in `config/default.yml`.

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -403,7 +403,7 @@ RSpec/ExampleWording:
   DisallowedExamples:
     - works
   VersionAdded: '1.0'
-  VersionChanged: '2.13'
+  VersionChanged: "<<next>>"
```
Comment (pirj) [inline config/default.yml:406]: This is not necessary, this is for cop configuration changes.

**Review by pirj — APPROVED** [review]
Thank you!

File: lib/rubocop/cop/rspec/example_wording.rb [hunk 2]
```diff
@@ -47,11 +50,13 @@ class ExampleWording < Base
         extend AutoCorrector
 
         MSG_SHOULD = 'Do not use should when describing your tests.'
+        MSG_WILL   = 'Do not use the future tense when describing your tests.'
         MSG_IT     = "Do not repeat 'it' when describing your tests."
         MSG_INSUFFICIENT_DESCRIPTION = 'Your example description is ' \
                                        'insufficient.'
 
         SHOULD_PREFIX = /\Ashould(?:n't)?\b/i.freeze
+        WILL_PREFIX   = /\Aw(?:ill|on't)\b/i.freeze
```
Comment (bquorning) [inline lib/rubocop/cop/rspec/example_wording.rb:59]: I think there’s nothing gained by isolating the `w`, but it becomes harder to read.
```suggestion
        WILL_PREFIX   = /\A(?:will|won't)\b/i.freeze
```

**Comment by jdufresne** [comment]
What is the project's preferred way to satisfy these offenses in the code? To me, breaking these small functions into smaller functions doesn't feel like a win. 

```
lib/rubocop/cop/rspec/example_wording.rb:70:9: C: Metrics/MethodLength: Method has too many lines. [12/10]
        def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler ...
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/rspec/wording.rb:20:7: C: Metrics/MethodLength: Method has too many lines. [12/10]
      def rewrite ...
      ^^^^^^^^^^^
```

**Comment by ydah** [comment]
> What is the project's preferred way to satisfy these offenses in the code? To me, breaking these small functions into smaller functions doesn't feel like a win.
> 
> ```
> lib/rubocop/cop/rspec/example_wording.rb:70:9: C: Metrics/MethodLength: Method has too many lines. [12/10]
>         def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler ...
>         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> lib/rubocop/rspec/wording.rb:20:7: C: Metrics/MethodLength: Method has too many lines. [12/10]
>       def rewrite ...
>       ^^^^^^^^^^^
> ```

You may want to silence the offense with a disable comment.　
refs: https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code

**Comment by jdufresne** [comment]
@ydah Thanks! I took this approach in the latest revision.

**Review by ydah — APPROVED** [review]
Looks good! Thank you.

**Pull request merged**